docs/standard/choosing-core-framework-server.md

```diff
@@ -93,7 +93,7 @@ In addition to the official roadmap, there are other frameworks to be ported to
 
 ### A need to use a platform that doesn’t support .NET Core
 
-Some Microsoft or third-party platforms don’t support .NET Core. For example, some Azure services such as Service Fabric Stateful Reliable Services and Service Fabric Reliable Actors require .NET Framework. Some other services provide an SDK not yet available for consumption on .NET Core. This is a transitional circumstance, as all of Azure services use .NET Core. In the meantime, you can always use the equivalent REST API instead of the client SDK.
+Some Microsoft or third-party platforms don’t support .NET Core. Some Azure services provide an SDK not yet available for consumption on .NET Core. This is a transitional circumstance, as all of Azure services use .NET Core. In the meantime, you can always use the equivalent REST API instead of the client SDK.
```

Comment:
@aljo-microsoft is still the case that some services still don't provide a SDK for consumption on .NET Core?

Reply:
@mairaw 
I don't know the validity of that statement, and hope that isn't the case still.

SF provides an SDK for our Reliable Services and Actors, and we support both in .Net Core and .Net Framework; So i'm only removing the invalid statement from the doc.

Reply:
I saw in the screenshot from the referenced issue. I was just trying to see if the rest of the sentence is also invalid at this point,
@cartermp would you know? 

Otherwise, I'm fine merging this as-is to at least remove the wrong SDKs being called out there as examples of services still using .NET Framework.

Reply:
Unfortunately, nuget.org doesn't show the target framework for each of these packages: https://www.nuget.org/profiles/servicefabric

But so long as they all target either netstandard2.0 or netcoreapp2.x, then this is good as-is.

Reply:
@mairaw,
can this be approved and merged?